lib/fastlane/actions/mailgun.rb

```diff
@@ -93,14 +86,37 @@ def self.handle_exceptions(options)
             Helper.log.fatal "Please provide a valid :message  = \"a_valid_mailgun_text\"".red
             raise 'No MAILGUN_MESSAGE given.'.red
           end
+
+          unless (options[:app_link] rescue nil)
+            Helper.log.fatal "Please provide a valid :app_link  = \"a_valid_mailgun_app_link\"".red
+            raise 'No MAILGUN_APP_LINK given.'.red
+          end
       end
 
-      def self.mailgunit(api_key,sandbox_domain,sandbox_postmaster,to,subject,text)
-        RestClient.post "https://api:#{api_key}@api.mailgun.net/v3/#{sandbox_domain}/messages",
-            :from => "Mailgun Sandbox<#{sandbox_postmaster}>",
-            :to => "#{to}",
-            :subject => subject,
-            :text => text
+      def self.mailgunit(options)
+        sandbox_domain = options[:postmaster].split("@").last
+        RestClient.post "https://api:#{options[:apikey]}@api.mailgun.net/v3/#{sandbox_domain}/messages",
+            :from => "Mailgun Sandbox<#{options[:postmaster]}>",
```

Comment:
Maybe we could also switch to the new syntax: `from: "Mailgun ..."` instead of `=>`